web/themes/custom/cloudy/src/components/alert/alert.twig

```diff
@@ -4,9 +4,9 @@
 {% set changed = changed|default(random(1262332800, 1577865600)) %} {# random timestamp between 2010 and 2020 #}
```

Comment:
Wouldn't this value get set in the demos under pattern lab rather than with a default that if missed would case fake data to be displayed?